2014-07-16 17:06:47.914 [error] <0.107.0>@basho_bench_worker:handle_info:149 Worker <0.108.0> exited with crash
2014-07-16 17:06:47.915 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_2 started with basho_bench_worker:start_link(basho_bench_worker_2, 2) at <0.107.0> exit with reason normal in context child_terminated
2014-07-16 17:06:48.062 [error] <0.94.0>@basho_bench_worker:handle_info:149 Worker <0.95.0> exited with crash
2014-07-16 17:06:48.063 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_1 started with basho_bench_worker:start_link(basho_bench_worker_1, 1) at <0.94.0> exit with reason normal in context child_terminated
2014-07-16 17:06:48.385 [error] <0.1487.0>@basho_bench_worker:handle_info:149 Worker <0.1488.0> exited with crash
2014-07-16 17:06:48.392 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_1 started with basho_bench_worker:start_link(basho_bench_worker_1, 1) at <0.1487.0> exit with reason normal in context child_terminated
2014-07-16 17:06:48.917 [error] emulator Error in process <0.1832.0> on node 'floppy_bench@127.0.0.1' with exit value: {{case_clause,{badrpc,{'EXIT',{function_clause,[{riak_dt_gcounter,update,[{add,3539},1,[]],[{file,"src/riak_dt_gcounter.erl"},{line,82}]},{materializer,update_snapshot,3,[{file,"src/materializer.erl"},{line,24}]},{floppy,read,2,[... 


2014-07-16 17:06:48.917 [error] <0.1831.0>@basho_bench_worker:handle_info:149 Worker <0.1832.0> exited with {{case_clause,{badrpc,{'EXIT',{function_clause,[{riak_dt_gcounter,update,[{add,3539},1,[]],[{file,"src/riak_dt_gcounter.erl"},{line,82}]},{materializer,update_snapshot,3,[{file,"src/materializer.erl"},{line,24}]},{floppy,read,2,[{file,"src/floppy.erl"},{line,31}]},{rpc,'-handle_call_call/6-fun-0-',5,[{file,"rpc.erl"},{line,205}]}]}}}},[{basho_bench_worker,worker_next_op,1,[{file,"src/basho_bench_worker.erl"},{line,244}]},{basho_bench_worker,max_worker_run_loop,1,[{file,"src/basho_bench_worker.erl"},{line,317}]}]}
2014-07-16 17:06:48.924 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_1 started with basho_bench_worker:start_link(basho_bench_worker_1, 1) at <0.1831.0> exit with reason normal in context child_terminated
2014-07-16 17:06:49.420 [error] <0.1344.0>@basho_bench_worker:handle_info:149 Worker <0.1345.0> exited with crash
2014-07-16 17:06:49.421 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_2 started with basho_bench_worker:start_link(basho_bench_worker_2, 2) at <0.1344.0> exit with reason normal in context child_terminated
2014-07-16 17:06:49.820 [error] <0.2398.0>@basho_bench_worker:handle_info:149 Worker <0.2400.0> exited with crash
2014-07-16 17:06:49.827 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_1 started with basho_bench_worker:start_link(basho_bench_worker_1, 1) at <0.2398.0> exit with reason normal in context child_terminated
2014-07-16 17:06:49.827 [error] <0.83.0> Supervisor basho_bench_sup had child basho_bench_worker_1 started with basho_bench_worker:start_link(basho_bench_worker_1, 1) at <0.2398.0> exit with reason reached_max_restart_intensity in context shutdown
